docs/data/data-grid/getting-started/migration-v5.md

```diff
@@ -0,0 +1,54 @@
+# Migration from v5 to v6
+
+<p class="description">This guide describes the changes needed to migrate from v5 to v6.</p>
+
+## Start using the alpha release
+
+In `package.json`, change the version of the data grid package to `next`.
+
+```diff
+-"@mui/x-data-grid": "latest",
++"@mui/x-data-grid": "next",
+```
+
+Using `next` ensures that it will always use the latest v6 alpha release, but you can also use a fixed version.
```

Comment:
```suggestion
Using `next` ensures that it will always use the latest v6 alpha release, but you can also use a fixed version like `6.0.0-alpha.0`.
```

To give example of alpha release version